add basic otelcol installation info for macOS (#1873)
SumoLogic/sumologic-documentation: docs/send-data/opentelemetry-distro-collector/install-collector-on-macos.md

sidebar_label: Install Collector on MacOS
description: Learn how to install Collector
---

> Follow the steps in this topic to install or uninstall an OpenTelemetry Collector on MacOS. See [OpenTelemetry Collector](/docs/send-data/opentelemetry-distro-collector) for information on other OSs.

Reviewer: ```suggestion
Follow the steps in this topic to install or uninstall an OpenTelemetry Collector on macOS. See [OpenTelemetry Collector](/docs/send-data/opentelemetry-distro-collector) for information on other operating systems.
```

## System Requirements​

TODO - add some info about system and hardware

> OpenTelemetry Collector is supported both on amd64 and arm64 architectures.

Reviewer: ```suggestion
The Sumo Logic OpenTelemetry Collector is supported both on amd64 and arm64 architectures.
```

## Install

> Install the OpenTelemetry Collector using either of the following methods:

Reviewer: ```suggestion
You can install our OpenTelemetry Collector using either of the following methods:
```

* [Install script](#install-script)
* [Manual step-by-step installation](#manual-step-by-step-installation)

### Install script

#### Get the installation token

Get your [installation token](https://help.sumologic.com/docs/manage/security/installation-tokens) if you don't have it already and assign it to environment variable:

```bash
export SUMOLOGIC_INSTALLATION_TOKEN=<TOKEN>
```

#### Run the installation script

You can run the script in two ways:

- by piping `curl` straight into `bash`:

   ```bash
   curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh | sudo -E bash -s -- -d
   ```

- by first downloading the script, inspecting its contents for security, and then running it:

   ```bash
   curl -o install-otelcol-sumo.sh https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh
   sudo -E bash ./install-otelcol-sumo.sh -d
   ```

The `-E` argument to `sudo` is needed to preserve the `SUMOLOGIC_INSTALLATION_TOKEN` environment variable in `sudo` session.

> The `-d` argument to the script is needed on MacOS. It will skip Systemd installation, which is not supported on MacOS.

Reviewer: ```suggestion
The `-d` argument to the script is needed on macOS. It will skip Systemd installation, which is not supported on macOS.
```

```bash
sudo bash ./install.sh -u
```

You can also use flag `-p` to remove all existing configurations as well:

```bash
sudo bash ./install.sh -u -p
```

### Manual step-by-step installation

If you installed the Collector manually, simply remove the binary from the directory you have placed it in:

```bash
sudo rm /usr/local/bin/otelcol-sumo 
```

## Upgrading the collector

### Upgrade OpenTelemetry Collector

First, you have to upgrade the Collector's version. The way you should do it, depends on how you installed it.

#### Install script

If you installed the Collector with the install script, you can use it to upgrade the Collector by using `-w` flag:

```bash
sudo bash ./install.sh -w -v 0.73.0-sumo-1
```

#### Manual step-by-step installation

If you installed the Collector manually, the simplest way to upgrade is to follow these steps:

- [Uninstall the Collector manually](#manual-step-by-step-installation-1)
- [Install the Collector again with a new version](#manual-step-by-step-installation)

### Update your config

After an upgrade, you should make sure that your config for OpenTelemetry Collector is up to date.

> To see changes in upstream OpenTelemetry components, please refer to the following changelogs:
> [core](https://github.com/open-telemetry/opentelemetry-collector/releases)
> and [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases).

Reviewer: ```suggestion
To see changes in upstream OpenTelemetry components, refer to the [core changelog](https://github.com/open-telemetry/opentelemetry-collector/releases) and [contrib changelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases).
```

List of breaking changes specific to Sumo Logic distribution of OpenTelemetry Collector can be found [here](https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/upgrading.md).

> ## Troubleshooting specific to MacOS

Reviewer: ```suggestion
## Troubleshooting specific to macOS
```
